### PR TITLE
ZK-4278: ZK animation uses "complete" callback. should use always ins…

### PR DESCRIPTION
**Pull request by plijyfes**
…tead

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -2694,6 +2694,7 @@ B86-ZK-4255.zul=A,E,Exception,ServerLog
 ##zats##B86-ZK-4241.zul=A,E,Popup,position
 ##zats##B86-ZK-4244.zul=A,E,Dropupload
 ##zats##B86-ZK-4270.zul=A,E,Tabbox,orient,Toolbar
+###zats#B86-ZK-4278.zul=A,E,anima,Animation
```
Comment (scribetw) [inline zktest/src/archive/test2/config.properties]: ##zats##

Comment (plijyfes) [reply zktest/src/archive/test2/config.properties]: updated

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**